ros2msg/ros2msg/api/__init__.py

```diff
@@ -65,7 +65,7 @@ def message_type_completer(**kwargs):
     for package_name, message_names in get_all_message_types().items():
         for message_name in message_names:
             message_types.append(
-                '{package_name}/{message_name}'.format_map(locals()))
+                '{package_name}/msg/{message_name}'.format_map(locals()))
```

Comment:
Maybe it's worth adding a comment that we're assuming all messages are in the subnamespace `msg`, which won't necessarily be true in the future. E.g. if we want to also show messages defined in `srv`, `action`, or an arbitrary namespace defined in an idl file.

Same for services below.

Reply:
I guess that remark applies to the majority of the message generation, resource index and usage pipeline. So I would rather not bother to add a comment in this specific case - especially since the explicit string literal states just that (imo).

Reply:
Fair enough.